sos/plugins/networking.py

```diff
@@ -17,10 +18,16 @@ class Networking(Plugin):
     plugin_name = "networking"
     profiles = ('network', 'hardware', 'system')
     trace_host = "www.example.com"
-    option_list = [(
+    option_list = [
         ("traceroute", "collect a traceroute to %s" % trace_host, "slow",
-         False)
-    )]
+         False),
+        ("namespaces", """string of specific namespaces regex to be
+          collected as \"eth* ens2\"""", "quick", ""),
```

Comment:
Use either "fast" or "" instead of "quick" (see e.g. https://github.com/sosreport/sos/blob/master/sos/plugins/dnf.py#L23 or https://github.com/sosreport/sos/blob/master/sos/plugins/postgresql.py#L32).

Maybe better to specify explicitly that the namespaces are separated by space (which would prolong the help text, though, so maybe remove the example? not decided..)

Reply:
changed to "fast", please check new commit